packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -503,6 +523,30 @@ const mmtParams = ref<MiraTemplateParams>({});
 
 const configuredMmt = ref(makeConfiguredMMT(mmt.value, knobs.value.transientModelConfig));
 
+const calendarSettings = computed(() => {
```

Comment:
I think this should be on its own component tera-calendar, as it can be used somewhere else.
But that can be done by the next person re-using this code.

Reply:
In my part 2 ticket I've added this as a util instead!